libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -184,6 +184,21 @@
         LOGIN_INSERTED_INVALID_URL,
         LOGIN_AUTOFILL_CREDENTIALS_FILLED,
         LOGIN_AUTOFILL_CREDENTIALS_UPDATED,
+        LOGIN_PROLOGUE_VIEWED,
+        LOGIN_EMAIL_FORM_VIEWED,
+        LOGIN_EMAIL_RETRY_VIEWED,
+        LOGIN_MAGIC_LINK_OPEN_FORM_VIEWED,
+        LOGIN_MAGIC_LINK_REQUEST_FORM_VIEWED,
+        LOGIN_PASSWORD_FORM_VIEWED,
+        LOGIN_PASSWORD_RETRY_VIEWED,
+        LOGIN_URL_FORM_VIEWED,
+        LOGIN_URL_RETRY_VIEWED,
```

Comment:
Hmm, not sure, what screen is this referring to? If it's the screen with the error "We couldn't find a WordPress website at that address.." then I think that would be the same screen as LOGIN_URL_FORM_VIEWED so, no new "VIEWED" event would be emitted.

OTOH, if you're thinking of adding a Track for the error event, then that would probably be with a different set of Tracks, possibly not a "VIEWED" one anyway.